packages/block-editor/src/components/block-tools/style.scss

```diff
@@ -84,84 +84,6 @@
 	}
 }
 
-/**
- * Block Label for Navigation/Selection Mode
- */
-
-.block-editor-block-list__block-selection-button {
-	display: inline-flex;
-	padding: 0 $grid-unit-15;
-	z-index: z-index(".block-editor-block-list__block-selection-button");
-
-	// Dark block UI appearance.
-	border-radius: $radius-small;
-	background-color: $gray-900;
```

Comment:
I think @richtabor wanted to try using the dark toolbar when in "content" mode

https://github.com/WordPress/gutenberg/issues/65298

I don't think we should allow this to block this PR - we can raise a followup.